### DISCUSSION TITLE
Canonical Correlation Analysis (CCA) Metric

### DISCUSSION THREAD

**Discussion by vlomonaco** [ContinualAI/avalanche#589] [Feature Request]
We need to add the CCA metric (described [here](https://arxiv.org/pdf/2004.09199.pdf)). Adding it only in Tensorboard would be fine I think.

**Comment by imprashr** [comment]
I almost finished this, just writing tests

**Comment by vlomonaco** [comment]
@imprashr is this finished? Since you already assigned yourself #1 maybe you can make a small and self-contained PR with this and let other people finish it?
